### DISCUSSION TITLE
Support for GitHub Enterprise as discussion backend?

### DISCUSSION THREAD

**Discussion by stefansli** [giscus/giscus#1589] [General]
Hi everybody,

at the moment it doesn't look like GitHub Enterprise is supported. However, after a quick code search, it looks like only a handful of changes need to be made to make it compatible. Is that something that would be welcome?

If so, how would you propose to configure the GitHub Enterprise server URL?
My first approach would be to use an environment variable as this is already used for the rest of the configuration. Does that sound good?

**Comment by laymonage** [comment]
Hello,

Yep that sounds good! I don't have access to a GHES instance so I can't test it, but if you can make the changes in such a way that it works for both GitHub.com and GHES that'd be great!

**Comment by radmacd** [comment]
Hey @stefansli, did you get anywhere with this? I am also running into an issue where I want to run this with GitHub Enterprise.

Thanks!
